css_styles/styles.css

```diff
@@ -587,3 +618,363 @@ footer {
 .footerbtns {
   display: flex;
 }
+
+/* Desktop version */
+
+@media screen and (min-width: 768px) {
+  .mymenu {
+    visibility: hidden;
+  }
+
+  #myheader {
+    background-color: #3e3c3c;
+  }
+
+  body {
+    overflow-x: hidden;
+  }
+
+  .nav-list {
+    display: flex;
+    margin-right: 206px;
+    margin-top: 26px;
+  }
+
+  .nav-list li {
+    list-style: none;
+    padding-right: 32px;
+    font-size: 15px;
+    font-weight: 600;
+    line-height: 20px;
+    font-family: "Inter", sans-serif;
+  }
+
+  .nav-list li a {
+    color: #fff;
+  }
+
+  .nav-list li a:hover {
+    color: grey;
+  }
+
+  .mylogo {
+    margin-left: 154px;
+  }
+
+  .mybiopage {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: start;
+    height: 870px;
+    width: 100%;
+    left: 0;
+    top: 0;
+    background-image: url(/images/Group-desktop.png);
+    background-repeat: no-repeat;
+    background-position: right;
+    background-color: #1c1a19;
+  }
+
+  .socialbtns-main {
+    height: 100%;
+    justify-items: center;
+    padding-top: 30px;
+  }
+
+  .socialbtns-main ul {
+    display: flex;
+    flex-direction: column;
+    height: 300px;
+  }
+
+  .mybio {
+    padding-left: 75px;
+    width: 1440px;
+    left: 0;
+    top: 0;
+  }
+
+  .mybio h1 {
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 78px;
+    line-height: 82px;
+    width: 470px;
+    height: 160px;
+    top: 192px;
+  }
+
+  .mybio h2 {
+    width: 566px;
+    height: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+  }
+
+  .mybio p {
+    width: 566px;
+    height: 123px;
+    font-family: "Inter", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  #myworks {
+    background-color: #fff;
+  }
+
+  .myworks {
+    grid-template-columns: 1fr 1fr;
+    padding: 0;
+    background-color: #fff;
+  }
+
+  #work-headline {
+    border: none;
+    line-height: 70px;
+  }
+
+  .line {
+    width: 55%;
+    border: 1px solid grey;
+    margin-top: 36px;
+  }
+
+  .multi-storybg {
+    height: 470px;
+    padding: 0;
+    margin: 0;
+    width: 50%;
+  }
+
+  .div-myworks {
+    height: auto;
+  }
+
+  #myworks-btn {
+    margin-top: 25px;
+  }
+
+  .projects-section {
+    grid-template-columns: 1fr 1fr 1fr;
+    background-color: #fff;
+  }
+
```

Comment:
> - [ ] While reviewing the changes you made as per the previous code reviewers' feedback, I noticed that the appropriate margin-bottom was not added, and the project sections were not aligned to match the Figma design. It is crucial to ensure that the design is as close to the Figma design as possible as it is one of the project requirements. Therefore, kindly make the necessary adjustments to ensure proper alignment and add the appropriate margin-bottom.😊
> 
> Figma design👍
> 
> ![margin-figma](https://user-images.githubusercontent.com/40009354/236642838-04ecbe82-f18e-4666-bc52-c747b3811467.png)
> 
> Your design👇:
> 
> ![margin](https://user-images.githubusercontent.com/40009354/236642850-8f6c1734-dd10-4ef7-ba38-a3ccf0db923c.png)
>

Reply:
I have updated and done the changes you mentioned sir. This is my last of the even extra reviews sir. Please approve my project so that I can advance forward. I really dont want to repeat the week sir. Thanks and best regards.